ethereum/solidity/src/test/QuantumGravityBridge.t.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.4;
+
+import "../lib/openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+
+import "../DataRootTuple.sol";
+import "../QuantumGravityBridge.sol";
+
+import "ds-test/test.sol";
+
+contract RelayerTest is DSTest {
+
+    // private keys used for hardcoded signatures signatures
+    // testAddr  = "0x9c2B12b5a07FC6D719Ed7646e5041A7E85758329"
+    // testPriv  = "64a1d6f0e760a8d62b4afdde4096f16f51b401eaaecc915740f71770ea76a8ad"
+    // testAddr2 = "0xe650B084f05C6194f6e552e3b9f08718Bc8a9d56"
+    // testPriv2 = "6e8bdfa979ab645b41c4d17cb1329b2a44684c82b61b1b060ea9b6e1c927a4f4"
+
+    bytes32 constant VALIDATOR_SET_HASH_DOMAIN_SEPARATOR =
+        0x636865636b706f696e7400000000000000000000000000000000000000000000;
+
+    bytes32 constant DATA_ROOT_TUPLE_ROOT_DOMAIN_SEPARATOR =
+        0x7472616e73616374696f6e426174636800000000000000000000000000000000;
```

Comment:
Are these domain separators chosen arbitrarily for this test?

Reply:
No, they're in the contract

https://github.com/celestiaorg/quantum-gravity-bridge/blob/cc0560976538fc3bde396e9d1fa4c7a9b1aefc75/ethereum/solidity/src/QuantumGravityBridge.sol#L43-L49